libpod/runtime.go

```diff
@@ -624,6 +624,25 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (retErr error) {
 		}
 	}
 
+	// Check current boot ID - will be written to the alive file.
+	systemBootID, err := os.ReadFile("/proc/sys/kernel/random/boot_id")
```

Comment:
I think this neeeds to be extracted to linux specific file as this file will not exists on freebsd

Reply:
I think we're OK, we don't actually report errors on this failing - we just don't proceed with comparing the boot ID. We could abstract it, but the only benefit would a few less syscalls on freebsd.

Reply:
yeah it would not result in errors but I think it would be better, we need to *remember* that this code must work even on freebsd. Maybe someone later decides hey this should handle the error and fail if we cannot read it...

Reply:
Fair